sbor/src/type_id.rs

```diff
@@ -155,6 +156,13 @@ impl<T: TypeId> TypeId for Rc<T> {
     }
 }
 
+impl<T: TypeId> TypeId for RefCell<T> {
```

Comment:
When is this needed for sbor?

Reply:
This is for `Rc<RefCell<ResourceContainer>>` in the vault.